build.gradle

```diff
@@ -451,6 +452,11 @@ task bundle(type: Zip) {
     from("gatk")
     from("README.md")
     from("build/docs/tabCompletion/gatk-completion.sh")
+
+    from("src/main/resources/org/broadinstitute/hellbender/utils/config/GATKConfig.properties") {
+        rename 'GATKConfig.properties', 'GATKConfig.EXAMPLE.properties'
```

Comment:
This is a definitely useful to have in our bundle.  I think we need to add something to A) the readme explaining how to use this, and B) probably in the example file itself.   Doesn't have to be part of this pr though.

Reply:
Agreed.  I'll add some info in the config file itself and I'll add a blurb in the Readme about it as well.